src/Lecture1.hs

```diff
@@ -123,4 +129,11 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: (Show a, Ord a) => a -> [a] -> String
+lowerAndGreater n list = go 0 0 list
+    where
+        go x y l
```

Comment:
You can write the type signature of `go` explicitly to fix the GHC warning about ambiguous types